Postman Test Suite/Ed-Fi ODS-API Profile Test Suite.postman_collection.json

```diff
@@ -9917,6 +9918,1640 @@
+													"];\r",
+													"\r",
+													"pm.test(\"The response model should contain the id and the primary key properties\",\r",
+													"    () => {\r",
+													"        _.each(keys, (key) => {\r",
+													"            pm.expect(originalEntity[key], key).to.deep.equals(profileEntity[key]);\r",
+													"        })\r",
+													"});\r",
+													"\r",
+													"pm.test(\"The response model should not contain the explicitly excluded properties\",\r",
+													"    () => {        \r",
+													"        // Traverse all properties that are not pripary keys\r",
+													"        for (const [key, value] of Object.entries(originalEntity).filter(function (e) { return this.indexOf(e[0]) < 0; }, keys)) \r",
+													"        {     \r",
+													"            // If property is not an array, make a deep comparison\r",
+													"            if (!Array.isArray(originalEntity[key])) {\r",
+													"                if(profileExcludeKeys.indexOf(key) > -1){\r",
+													"                    pm.expect(profileEntity[key], key).to.be.undefined;\r",
+													"                }\r",
+													"                else {\r",
+													"                    pm.expect(profileEntity[key], key).to.deep.equal(originalEntity[key]);\r",
+													"                }\r",
+													"            }\r",
+													"            // if property is an array, exclude empty arrays\r",
+													"            else if (originalEntity[key].length !== 0){\r",
+													"                if(profileExcludeKeys.indexOf(key) > -1){\r",
+													"                    pm.expect(profileEntity[key], key).to.be.undefined;\r",
+													"                }\r",
+													"            }\r",
+													"        }  \r",
+													"});\r",
+													""
+												],
+												"type": "text/javascript"
+											}
+										}
+									],
+									"request": {
+										"method": "GET",
+										"header": [
+											{
+												"key": "Accept",
+												"value": "application/vnd.ed-fi.studentspecialeducationprogramassociation.studentspecialeducationprogramassociation-derived-association-excludeonly.readable+json",
+												"type": "text"
+											}
+										],
+										"url": {
+											"raw": "{{ApiBaseUrl}}/data/v3/ed-fi/studentSpecialEducationProgramAssociations/{{known:studentSpecialEducationProgramAssociationId}}",
+											"host": [
+												"{{ApiBaseUrl}}"
+											],
+											"path": [
+												"data",
+												"v3",
+												"ed-fi",
+												"studentSpecialEducationProgramAssociations",
+												"{{known:studentSpecialEducationProgramAssociationId}}"
+											]
+										}
+									},
+									"response": []
+								}
+							]
+						},
+						{
+							"name": "Write Requests",
+							"item": [
+								{
+									"name": "The Write content type only excludes certain properties",
```

Comment:
check assertions "the persisted entity model should not have the new values assigned to the explicitly excluded properties" and "the only values not changed should be the explicitly excluded values, the id, and the primary key values"